packages/foundry/scripts-js/revealPK.js

```diff
@@ -0,0 +1,37 @@
+import { listKeystores } from "./listKeystores.js";
+import { execSync } from "child_process";
+
+async function revealPk() {
+  try {
+    console.log("👀 This will reveal your private key on the console.");
+
+    const selectedKeystore = await listKeystores(
+      "Select a keystore to reveal its private key: "
```

Comment:
nit: it's not clear that user should write keystore number, not name

Reply:
Fixed it https://github.com/scaffold-eth/scaffold-eth-2/pull/1094/commits/52181ff140db4065e6fb1814dcac49a36d933b70 added an example and made it more explicit 🙌

Reply:
great, thanks!